utils.js

```diff
@@ -21,6 +21,21 @@ let chakram = require('chakram');
  */
 let utils = {
     /**
+     * Ensure that a Chakram response object is sane before using it.
+     * Throws if the response is unreasonable.
+     *
+     * @param {ChakramResponse} response Object to check.
+     */
+    assertSaneResponse: function assertSaneResponse(response) {
+        if (!response) {
+            throw new Error('Undefined response received!');
```

Comment:
I actually need to change this, I shouldn't say something is undefined if it could be some other falsy value.